ledfx/effects/texter2d.py

```diff
@@ -131,6 +131,11 @@ class Texter2d(Twod, GradientEffect):
                 description="general speed slider for text effects",
                 default=1,
             ): vol.All(vol.Coerce(float), vol.Range(min=0.0, max=3)),
+            vol.Optional(
```

Comment:
Call this left_scroll and default to true.

It's good to have default as true setting and left_scroll is about as sane as we can get right now

Reply:
what about adding something like this? 
![image](https://github.com/user-attachments/assets/dc44f79d-168a-424c-b5ce-7fd296713d40)

Reply:
It's a coin toss at this point, boolean sounds simpler, but picker is more descriptive. I think I like your picker more.